README.md

```diff
@@ -0,0 +1,90 @@
+# browsertrix-harvester
+
+CLI app for performing a web crawl via [Browsertrix-Crawler](https://github.com/webrecorder/browsertrix-crawler), and optionally parsing structured metadata records from the crawl output [WACZ](https://replayweb.page/docs/wacz-format) file.
+
+## Architecture
+
+See [architecture docs](docs/architecture.md).
+
+## Development
+
+**NOTE**: When performing web crawls, this application invokes browsertrix-crawler.  While theoretically possible to install browsertrix-crawler on your local machine as a callable binary, this application is oriented around running only inside of a Docker container where it is already installed.  For this reason, the pipenv convenience command `harvest-dockerized` has been created.
+
+## Environment Variables
+
+```dotenv
+WORKSPACE=None
+SENTRY_DSN=None# If set to a valid Sentry DSN, enables Sentry exception monitoring. This is not needed for local development.
+```
+
+### Build Application
+
+- To install with dev dependencies: `make install`
+- To update dependencies: `make update`
+- To run unit tests: `make test`
+- To lint the repo: `make lint`
+- Install pre-commit hooks: `pipenv run pre-commit install`
```

Comment:
Hmm, in another repo, I organized the `pipenv run pre-commit install` command as part of `make install`: 

```
install: # install python dependencies
	pipenv install --dev
	pipenv run pre-commit install
```

I think my reason for doing it was that it would be less likely to be missed and prioritizes installation of the pre-commit hooks and having the `--no-verify` option be the way to skip this process! Just a thought.

Reply:
Good call!  Will make this update.